src/StyleableBannerWithLink/index.tsx

```diff
@@ -74,14 +81,32 @@ const StyleableBannerWithLink: React.FC<Props> = (props: Props) => {
             buttonText,
             buttonUrl,
             showPaymentIcons = 'false',
+            reminderStage,
+            reminderOption,
             imageUrl,
             imageAltText = '',
             imagePosition = 'center',
         },
         trackClick,
+        fetchEmail,
     } = props;
 
-    const styles = selfServeStyles(props.brazeMessageProps, defaultColors);
+    const brazeProps = props.brazeMessageProps;
+
+    const styles = selfServeStyles(brazeProps, defaultColors);
+
+    const reminderStyles = {
+        styleReminderButton: brazeProps?.styleReminderButton as ColorValueHex,
+        styleReminderButtonBackground: brazeProps?.styleReminderButtonBackground as ColorValueHex,
+        styleReminderButtonHover: brazeProps?.styleReminderButtonHover as ColorValueHex,
+        styleReminderAnimation: brazeProps?.styleReminderAnimation as ColorValueHex,
```

Comment:
I'm confused about the casting happening here. These are all optional according to the `BrazeMessageProps` type above, but here we're casting to `ColorValueHex` without defaulting. That feels like it might cause problems downstream where the values maybe undefined at runtime but TypeScript won't know to warn us about this because we've said here they won't be (`ColorValueHex` is defined as a string template).

Reply:
I'll experiment and reach out if (when) it starts taking too long to rectify

Reply:
Fixed (I think)